modules/statistics/src/main/frontend/src/Statistics/AdminStatistics.jsx

```diff
@@ -0,0 +1,311 @@
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+import React, { useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { v4 as uuidv4 } from 'uuid';
+import { 
+  Button, 
+  Card, 
+  CardContent,
+  CardHeader,
+  Dialog, 
+  DialogActions, 
+  DialogContent, 
+  DialogTitle,
+  Grid,
+  IconButton,
+  Tooltip,
+  Typography,
+  withStyles,
+} from "@material-ui/core";
+import statisticsStyle from "./statisticsStyle.jsx";
+import NewItemComponent from "../components/NewItemButton.jsx";
+import ResponsiveDialog from "../components/ResponsiveDialog.jsx";
+import LiveTable from "../dataHomepage/LiveTable.jsx";
+import DeleteButton from "../dataHomepage/DeleteButton.jsx";
+import Fields from "../questionnaireEditor/Fields.jsx";
+import EditIcon from "@material-ui/icons/Edit";
+import { formatIdentifier } from "../questionnaireEditor/EditorInput.jsx";
+import { fetchWithReLogin, GlobalLoginContext } from "../login/loginDialogue.js";
+
+/**
+ * Createa the LIveTable cell contents for a given node. This generates a link to the
+ * given node (via its admin page), or returns the node's label if no valid link can be made.
+ *
+ * @param {Object} node The lfs:SubjectType or lfs:Question node to generate a link for
+ */
+function CreateTableCell(node) {
```

Comment:
This is not used as a React component, so I think it should not start with a capital letter:

```suggestion
function createTableCell(node) {
```